omi/firmware/firmware/src/lib/battery/battery.c

```diff
@@ -183,6 +183,14 @@ int battery_get_millivolt(uint16_t *battery_millivolt)
 
 int battery_get_percentage(uint8_t *battery_percentage, uint16_t battery_millivolt)
 {
+    // Fix for issue #1885: Battery never shows 100% even when fully charged
```

Comment:
@beastoin Here is a comment regarding this number. 
Omi never charges till 4200, which equal to 100%